rahtikirja-tulostus.php

```diff
@@ -853,7 +878,7 @@
       if (!isset($tee_varsinainen_tulostus) or (isset($tee_varsinainen_tulostus) and $tee_varsinainen_tulostus)) {
 
         // tulostetaan toimitustavan m��rittelem� rahtikirja
-        if (@include "tilauskasittely/$toitarow[rahtikirja]") {
+        if (($_onko_unifaun and $_tulostustapa and $_paktiedot) or @include "tilauskasittely/$toitarow[rahtikirja]") {
```

Comment:
@tonyselkamo @samiluukkonen 

Mikä tässä on ollut ajatus?

if (($_onko_unifaun and $_tulostustapa and $_paktiedot) or @include "tilauskasittely/$toitarow[rahtikirja]") {

Mikä on se keissi, että toi "or @include "tilauskasittely/$toitarow[rahtikirja]""-ehto ei toteutuisi?